app/browser/api/ledger.js

```diff
@@ -2160,7 +2167,13 @@ const transitionWalletToBat = () => {
   }
 
   if (client.busyP()) {
-    console.log('ledger client is currently busy; transition will be retried on next launch')
+    if (++busyRetryCount > 3) {
+      console.log('ledger client is currently busy; transition will be retried on next launch')
+      return
+    }
+    const delayTime = random.randomInt({ min: 1, max: 500 })
```

Comment:
I would add higher values, because now we are setting timeout between 1ms and 500ms

Reply:
Fixed 👍